tools/twitter-api/get-tweets.py

```diff
@@ -0,0 +1,98 @@
+from elasticsearch import Elasticsearch
+from random import randint
+import json
+import tweepy
+
+creds = json.load(open("creds.json"))
+creds = creds["twitter-creds"]
+
+consumer_key        = creds["consumer_key"]
+consumer_key_secret = creds["consumer_key_secret"]
+access_token        = creds["access_token"]
+access_token_secret = creds["access_token_secret"]
+
+auth = tweepy.OAuthHandler(consumer_key, consumer_key_secret)
+auth.set_access_token(access_token, access_token_secret)
+api = tweepy.API(auth)
+
+def GetElasticSearchHits(): 
+  es = Elasticsearch(
+    ['lp01.idea.rpi.edu:443/elasticsearch'],
+    # turn on SSL
+    use_ssl=True,
+    # no verify SSL certificates
+    verify_certs=False,
+    # don't show warnings about ssl certs verification
+    ssl_show_warn=False
+  )
+   
+  # create a Python dictionary for the search query:
+  search_param = {
+    "_source": True,
+    "query": {
+      "constant_score" : {
+           "filter" : {
+              "exists" : {
+                 "field" : "in_reply_to_status_id_str"
+              }
+           }
+      }
+    }
+  }
+   
+  # print out searchable indexes in elasticsearch
```

Comment:
If this file is being printed just to see what the index fields are, Kibana can show the same view. I've requested you guys get access to Kibana.